server/src/user-house/user-house.service.ts

```diff
@@ -129,11 +129,6 @@ export class UserHouseService {
         );
       }
 
-      // Check if the user is an admin
-      if (!operatingUserHouse.isAdmin) {
-        throw new ForbiddenException('Only admins can update admin status.');
-      }
-
```

Comment:
I have a question!  Can a non-admin user delete the other users? We should have validation here too (Only the validation in client side is not enough)

Reply:
No. Non-admin users can't remove the house members because they can't see the 'members management' modal. We need to remove this validation. Otherwise, non-admin users can't leave a house on the house-info page.

Reply:
JS can be changed in the browser, so relying on just client-side validation is not enough. (In this case, usually people don't see the member list modal but you can change it). So here, you need the validation that if the user is not the admin and trying to delete the other uses, send an error. Could you add it?

Reply:
Got it! I added the validation. 
c57cbc7a84ccc197b184af5057e184d8fee5953b
bd98b6738ac845d3fa5dddc1611cc5cfe2426402